### PR TITLE
Add CLI blackbox tests

### PR DESCRIPTION
**Pull request by jbaublitz**
Related to #510 

One note for @mulkieran: I added a test to ensure that the cache is mutually exclusive with encryption but I can remove it if that doesn't add any additional value or if the blackbox tests should not be checking this.

@bgurney-rh I was unable to test these changes from my machine so would you be able to run these after I get confirmation from @mulkieran that both added tests should be there?

### CODE REVIEW

**Comment by mulkieran** [comment]
@bgurney-rh Do you want to go ahead and test this? It all looks good to me.

File: tests/blackbox/stratis_cli_cert.py [hunk 1]
```diff
@@ -148,6 +162,42 @@ def test_pool_create(self):
...
+        Test creating an encrypted pool with cache.
+
+        This should fail.
+        """
+        with KernelKey("test-password") as key_desc:
+            pool_name = make_test_pool(StratisCertify.DISKS[0:2], key_desc)
+            self.unittest_command(
+                [STRATIS_CLI, "pool", "init-cache", pool_name, StratisCertify.DISKS[2]],
+                1,
+                True,
```
Comment (mulkieran) [inline tests/blackbox/stratis_cli_cert.py:197]: For some reason stderr comes before standout in arg order and I think it's stderr that we should expect to be non-empty.

Comment (jbaublitz) [reply tests/blackbox/stratis_cli_cert.py:197]: Okay I assumed it would be the opposite! I'll fix that.

Comment (jbaublitz) [reply tests/blackbox/stratis_cli_cert.py]: Should be fixed now.

**Comment by bgurney-rh** [comment]
@jbaublitz after trying commits 55ef10 and 78bb53, all blackbox tests pass.

**Pull request closed without merging**